apps/daimo-mobile/src/view/screen/HistoryOpScreen.tsx

```diff
@@ -66,12 +76,17 @@ function HistoryOpScreenInner({ account, op }: Props) {
 
   const { chainConfig } = env(daimoChainFromId(account.homeChainId));
 
+  const nav = useNav();
+
   return (
     <View style={ss.container.screen}>
       <ScreenHeader
         title="Transfer"
         onExit={() => {
-          toggleBottomSheet(false); // Collapse to small height
+          if (nav.canGoBack()) {
+            changeBottomSheetSnaps(3);
```

Comment:
it feels weird for the HistoryOpScreen (inner component) to have to change the SwipeUpDown (outer component) this way

if i understand correctly:
- SwipeUpDown always has 2 snap points (~25% open and 100% open)
- in some situations, it has a third (half screen)
- `changeBottomSheetSnaps` is how we're toggling this

Reply:
yep that's correct -- when on the list screen we want to show three snap points with the lowest being the default dismissed version, when on the detail op screen we want to show two snap points with the lower being the middle "active" (with changed background) version

Reply:
not super happy with the ugliness either but best I could come up with.